package.json

```diff
@@ -284,6 +284,7 @@
     "debounce-stream": "^2.0.0",
     "deep-freeze-strict": "1.1.1",
     "end-of-stream": "^1.4.4",
+    "enzyme": "^3.11.0",
```

Comment:
You will need to rewrite all tests using React testing library, as enzyme was recently removed and we do not want to backtrack on that effort.